tests/test_validate.py

```diff
@@ -46,7 +46,7 @@ def test_ui_valid_bundle_io(bundle, expected_validation_results_dict):
             {'errors': [
                 "csv.spec.links must be a list of name & url pairs.",
                 "spec.version invalid is not a valid version "
-                "(example of a valid version is: v1.0.12)",
+                "(example of a valid version is: 1.0.12)",
```

Comment:
Can we make sure to have both a positive and a negative test?

Reply:
I'll use `v<anything>` as the negative test for this field

Reply:
Actually, since this is already the negative, do you mean we need a positive test case?

Reply:
For the positive test case, isn't the `valid.bundle.yaml` sufficient? Or should I test additional valid semvers? I don't think I need to test this since we're using an external package.